pinot-controller/src/test/java/org/apache/pinot/controller/helix/PinotControllerModeTest.java

```diff
@@ -248,6 +253,16 @@ public void testPinotOnlyController() {
     helixOnlyController.stop();
   }
 
+  private void checkHelixConstraints(HelixAdmin helixAdmin) {
+    ClusterConstraints constraints =
+        helixAdmin.getConstraints(getHelixClusterName(), ClusterConstraints.ConstraintType.MESSAGE_CONSTRAINT);
+    ConstraintItem item = constraints.getConstraintItem("MaxStateTransitionsPerInstance");
+    Assert.assertEquals(".*", item.getAttributeValue(ClusterConstraints.ConstraintAttribute.INSTANCE));
```

Comment:
Please reverse the arguments for all the asserts, where the first should be actual, and the second should be expected.